packages/kbn-i18n/package.json

```diff
@@ -26,7 +26,7 @@
     "@types/json5": "^0.0.30",
     "@types/react-intl": "^2.3.11",
     "cross-env": "^5.2.0",
```

Comment:
Bump'd here since it requires `event-stream`, and is now fixed in `4.1.5`

Reply:
This same bump is happening here: https://github.com/elastic/kibana/pull/26200